### PR TITLE
Generate Shark Tank for tflite

### PR DESCRIPTION
**Pull request by AmosLewis**
Add an generate_shark_tank to download all the model file into tmp local directory. Then you can upload it by gsutil. Right now, it only work for tflite model download. 
 python generate_sharktank.py --tflite_model_list="./tank/tflite/tflite_model_list.csv"

### CODE REVIEW

File: generate_sharktank.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        if self.torch_model_list is not None:
+            print("Process torch model")
+        if self.tf_model_list is not None:
+            print("Process torch model")
+
+        print("self.tflite_model_list: ", self.tflite_model_list)
+        # compile and run tfhub tflite
+        if self.tflite_model_list is not None:
+            print("Setting up for tflite TMP_DIR")
+            self.tflite_workdir = os.path.join(os.path.dirname(__file__), "./tmp_shark_tank/tflite")
```
Comment (powderluv) [inline generate_sharktank.py:30]: gen_shark_tank/

File: generate_sharktank.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                            import_only=False)
+
+        if self.upload == True:
+            print("upload tmp tank to gcp")
+            os.system('gsutil cp -r ./gen_shark_tank gs://shark_tank/')
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--torch_model_list", type=str, default="./tank/torch/torch_model_list.csv")
+    parser.add_argument("--tf_model_list", type=str, default="./tank/tf/tf_model_list.csv")
+    parser.add_argument("--tflite_model_list", type=str, default="./tank/tflite/tflite_model_list.csv")
```
Comment (raikonenfnu) [inline generate_sharktank.py:75]: Is there a purpose of the black_list.csv? Looks like it is not being used here.

File: generate_sharktank.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+                    if os.path.exists(iree_ir):
+                        print(iree_ir, "exists")
+                    else:
+                        print("Convert tflite to tosa.mlir")
+                        ireec_tflite.compile_file(
+                            tflite_saving_file,
+                            input_type="tosa",
+                            save_temp_iree_input=iree_ir,
+                            target_backends=[IREE_TARGET_MAP['cpu']],
```
Comment (raikonenfnu) [inline generate_sharktank.py:65]: do we need to set target_backend for compile_file?

**Review by powderluv — APPROVED** [review]

File: generate_sharktank.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                 upload: bool = False):
+        self.torch_model_list = torch_model_list
+        self.tf_model_list = tf_model_list
+        self.tflite_model_list = tflite_model_list
+        self.upload = upload
+
+        if self.torch_model_list is not None:
+            print("Process torch model")
+        if self.tf_model_list is not None:
+            print("Process torch model")
```
Comment (raikonenfnu) [inline generate_sharktank.py:26]: Can we add a "TORCH HUB NOT IMPLEMENTED YET.", and "TF HUB NOT IMPLEMENTED YET."

**Comment by powderluv** [comment]
Please address stan's coments

File: generate_sharktank.py (hunk 3 above)
Comment (AmosLewis) [reply generate_sharktank.py:65]: yes. I tried to delete it, but cause some bug.

**Pull request merged**